conversion-script.sh

```diff
@@ -93,6 +93,6 @@ for file in pictures-svg/*.svg; do
         # as not all SVG are updated to have no borders, trim unnecessary white borders from png
         mogrify -trim "pictures-png/$normalized.png"
         # even though mogrify trims the png, we need to trim again for the jpg
-        convert "pictures-png/$normalized.png" -background white -flatten -alpha off -trim "pictures-jpg/$normalized.jpg"
+        convert "pictures-png/$normalized.png" -resize 256x256 -background white -flatten -alpha off -trim "pictures-jpg/$normalized.jpg"
```

Comment:
Can you only specify a max height or width? Otherwise you probably squeeze images that are either landscape or portrait.

Reply:
From what I have seen, this looks fine:
https://map.aachen.freifunk.net/pictures-jpg/ubiquiti-unifi-ac-mesh.jpg

I deployed the pictures in this folder that way:
https://map.aachen.freifunk.net/pictures-jpg/

Reply:
https://imagemagick.org/Usage/resize/

Good to know!
I guess you also want to add "/>" at the end? 
Alternatively, if you don't care about dimension and only about file size, the `@` operator seems a better choice for images with a very strong aspect ratio